Library/Homebrew/brew.sh

```diff
@@ -120,6 +120,11 @@ case "$*" in
     source "${HOMEBREW_LIBRARY}/Homebrew/formula_path.sh"
     homebrew-formula-path "$@" && exit 0
     ;;
+  # falls back to cmd/command.rb on a non-zero return
+  command*)
```

Comment:
```suggestion
  command)
```
Wouldn't this allow you to get rid of the check for `commandsomething` in `homebrew-command-path`?

Reply:
@carlocab Sadly this breaks if any arguments are passed because it's on `"$*"` and not  the first argument